docs/tutorials/t-08.md

```diff
@@ -72,7 +72,12 @@ We use `connectAlgoSigner()` method to connect the AlgoSigner wallet. Next, we c
 ```js
    async connectMyAlgoWallet() {
       try {
-        let myAlgo = new MyAlgoWalletSession(ChainType.MainNet);
+        let walletURL = {
```

Comment:
you can add the type here

Reply:
Can we add type to this? Because walletURL is now an object in javascript

Reply:
Yes, we can. let use `interface`. TS support it very well.

Reply:
Ah, My bad. This example in js not ts.